src/App.js

```diff
@@ -83,7 +83,7 @@ class App extends React.Component {
   render() {
```

Comment:
- Aaron, great job on competing you Todo App. Great work adding search functionality and validation for empty todos. 
- Your git workflow is fluid and You did amazing job on this assingment 👍